src/sentry/coreapi.py

```diff
@@ -382,24 +394,24 @@ def should_filter(self, project, data, ip_address=None):
         so that we can store it in metrics
         """
         if ip_address and not is_valid_ip(project, ip_address):
-            return (True, 'ip_address')
+            return (True, FilterStatKeys.IP_ADDRESS)
 
         release = data.get('release')
         if release and not is_valid_release(project, release):
-            return (True, 'release_version')
+            return (True, FilterStatKeys.RELEASE_VERSION)
 
         message_interface = data.get('sentry.interfaces.Message', {})
         error_message = message_interface.get('formatted', ''
                                               ) or message_interface.get('message', '')
         if error_message and not is_valid_error_message(project, error_message):
-            return (True, 'error_message')
+            return (True, FilterStatKeys.ERROR_MESSAGE)
 
         for filter_cls in filters.all():
             filter_obj = filter_cls(project)
             if filter_obj.is_enabled() and filter_obj.test(data):
-                return (True, 'other_filter')
+                return (True, six.text_type(filter_obj.id))
```

Comment:
I still think this is a little weird but I don't have any better ideas offhand.

Reply:
Yeah I agree it's not exactly intuitive, but hopefully using the `FilterStatsKeys` enum for the`Filter.id` will keep values consistent.